### DISCUSSION TITLE
Next.js example

### DISCUSSION THREAD

**Discussion by brillout** [hattipjs/hattip#173] [General]
Using [Next.js's edge runtime](https://nextjs.org/docs/api-reference/edge-runtime), is it possible to intercept Next.js's requests and serve them with HatTip instead? In that case it may be possible to use Next.js with HatTip.

@QuiiBz Fatih told me you managed to deploy Next.js to Lagon. Did you directly use Next.js's edge runtime or did you intercept requests?

**Comment by QuiiBz** [comment]
I'm using `vercel build` to get a bundled output of each page/api route, then I manually update the code to match the handler's signature with the one Lagon requires. I can then run this bundle using Lagon's edge runtime.

As you see it currently requires some manual steps, which I'm not sure yet how to automate: https://github.com/lagonapp/lagon/issues/47
